demo/demo.less

```diff
@@ -33,6 +33,14 @@
   line-height: 36px;
 }
 
+.mdl-button--icon {
```

Comment:
This duplicates what you have below, where you do the same for both selectors:

```css
.mdl-button--fab,
.mdl-button--icon {
```

You can delete this one.

Reply:
done!